src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -340,29 +340,29 @@ void insertTrack(
     QString anlzPath = devicePath + getText(track->analyze_path());
     mixxx::RgbColor::optional_t color = mixxx::RgbColor::nullopt();
 
-    switch (track->color_id()) {
-    case TRACK_COLOR_PINK:
+    switch (static_cast<TrackColor>(track->color_id())) {
```

Comment:
I think you should make the enum class of the same base type as the return value of track->color_id()

Reply:
Also having some issues with this, TrackColor enum is now defined as:
`enum class TrackColor : uint8_t {`
And compiler is reporting:
`src/library/rekordbox/rekordboxfeature.cpp:365:10: error: value of type '(anonymous namespace)::TrackColor' is not implicitly convertible to 'int'`

Reply:
It appears an `enum class` can't be implicitlty converted to any other type, including the base type: https://stackoverflow.com/a/14968068

Reply:
I don't get what you need implicit conversions for...

This compiles just fine for me:
```diff
diff --git a/src/library/rekordbox/rekordboxfeature.cpp b/src/library/rekordbox/rekordboxfeature.cpp
index a79dc098ce..a3ce4562ff 100644
--- a/src/library/rekordbox/rekordboxfeature.cpp
+++ b/src/library/rekordbox/rekordboxfeature.cpp
@@ -48,7 +48,7 @@ const QString kPdbPath = QStringLiteral("PIONEER/rekordbox/export.pdb");
 const QString kPLaylistPathDelimiter = QStringLiteral("-->");
 const double kLongestPosition = 999999999.0;

-enum class TrackColor {
+enum class TrackColor : uint8_t {
     TrackColorPink = 1,
     TrackColorRed,
     TrackColorOrange,
```

Reply:
My mistake, my assumption was that this comment was suggesting removing the static cast in the switch statement. All compiling now with static cast in the switch.